frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsAndSoil.tsx

```diff
@@ -106,15 +110,11 @@ const FieldsAndSoilComponent: React.FC<InputModuleProps> = ({
     <>
       {isSubmitted && (
         <>
-          <FieldsListComponent
-            farmDetails={farmDetails}
-            updateFarmDetails={updateFarmDetails}
-            handleFormState={handleFormState}
-          />
+          <FieldsListComponent farmDetails={farmDetails} />
           {!isFieldAdded && (
             <FieldsButtonComponent
               addNewField={addNewField}
-              submitFarmInfo={submitFarmInfo}
+              updateFarmDetails={() => updateFarmDetails(farmDetails)}
```

Comment:
this works fine, but you can also do `updateFarmDetails.bind(this, farmDetails)` instead of the anonymous function.
More of a "the more you know" kind of thing, not necessary

Reply:
I love this kind of tip, thank you :)